### PR TITLE
Feature/5803 super component builder

### PR DESCRIPTION
**Pull request by s3341458**

### CODE REVIEW

File: python/users.py [hunk 1]
```diff
@@ -17,6 +17,19 @@
 from sdk.python.domains import EnrolledDomain
 
 
+class SystemRole(sdk.python.entities.Entity):
```
Comment (DanielCollins) [inline python/users.py]: should maybe have it's own file?

File: python/users.py [hunk 2]
```diff
@@ -17,6 +17,19 @@
...
+
+    resource = '/system_roles/'
+    json_name = 'system_role'
+
+    def __init__(self, **kwargs):
+        super(SystemRole, self).__init__(**kwargs)
```
Comment (DanielCollins) [inline python/users.py]: i think we can just have `super()`... in python3

File: python/util/view_access_level.py [hunk 3]
```diff
@@ -2,4 +2,5 @@
 IN_DOMAIN_CONTEXT = 1
 LOGGED_IN_USER = 2
 MANAGED_DOMAIN = 3
-SUPER_USER_ONLY = 4
```
Comment (DanielCollins) [inline python/util/view_access_level.py]: better not to edit these kinds of constants -- just make new ones.

File: python/users.py (hunk 1 above)
Comment (s3341458) [reply python/users.py]: I think put them in file is better, they are ... too close

File: python/users.py (hunk 2 above)
Comment (s3341458) [reply python/users.py]: agreed

File: python/util/view_access_level.py (hunk 3 above)
Comment (s3341458) [reply python/util/view_access_level.py]: agreed

**Comment by s3341458** [comment]
will fix tonight

**Comment by s3341458** [comment]
ready for another look

File: python/users.py (hunk 1 above)
Comment (DanielCollins) [reply python/users.py]: it does make it more difficult to find the definition.

**Pull request merged**